modules/virt-attaching-vm-secondary-network-cli.adoc

```diff
@@ -37,17 +37,20 @@ spec:
       networks:
         - name: default
           pod: {}
-        - name: bridge-net <1>
+        - name: bridge-net <2>
           multus:
-            networkName: <network-namespace/a-bridge-network> <2>
+            networkName:  <network-namespace>/<a-bridge-network> <3>
 ...
 ----
-<1> The name of the bridge interface. This value must match the `name` value of the corresponding `spec.template.spec.networks` entry.
-<2> The name of the network attachment definition.
+<1> The name of the bridge interface.
+<2> The name of the bridge interface. This value must match the `name` value of the corresponding `spec.template.spec.networks` entry.
+<3> The namespace where the Network Attachment Definition exists. The only 2 possible options for this namespace are either the default namespace or the same namespace where the VM is to be created.
```

Comment:
Callout <3> marks the entire line, but the callout contents now refers only to the namespace part. I would change **only the first sentence** to something like:
_The name of the Network Attachment Definition, prefixed by the namespace where it exists._
(Sorry I didn't catch it earlier, it only occurred to me now).

Reply:
Ok, let's try again